src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -114,6 +113,91 @@
+     * This can be overridden by supplying a list of node pairings to reanchor relationships in the subgraph clone to other nodes in the graph.
+     * This is useful when instead of cloning a certain node or set of nodes, you want to instead reanchor relationships in the resulting clone
+     * such that they point to some different existing node in the graph.
+     *
+     * For example, this could be used to clone a branch from a tree structure (with none of the new relationships going
+     * to the original nodes) and to reanchor any relationships from an old root node (which will not be cloned) to a different existing root node.
+     *
+     */
+    @Procedure(mode = Mode.WRITE)
+    @Deprecated
+    @Description("apoc.refactor.cloneSubgraph([node1,node2,...], [rel1,rel2,...], [reanchorNodePairs = [] ], [skipProperties = [] ]) YIELD input, output, error | " +
+            "clone nodes with their labels and properties (optionally skipping specific properties), and clone the given relationships (will exist between cloned nodes only or be reanchored according to reanchor node pairings)")
+    public Stream<NodeRefactorResult> cloneSubgraph(@Name("nodes") List<Node> nodes,
+                                                    @Name("rels") List<Relationship> rels,
+                                                    @Name(value = "reanchors", defaultValue = "[]") List<List<Node>> reanchors,
+                                                    @Name(value = "skipProperties", defaultValue = "[]") List<String> skipProperties) {
+        Map<Node, Node> copyMap = new LinkedHashMap<>(nodes.size());
+        Map<Node, Node> reanchorMap = reanchors.isEmpty() ? Collections.emptyMap() : new LinkedHashMap<>(reanchors.size());
+        List<NodeRefactorResult> resultStream = new ArrayList<>();
+        if (nodes == null) return Stream.empty();
+
+        // populate reanchor map
+        for (List<Node> pairing : reanchors) {
+            if (pairing == null) continue;
+
+            if (pairing.size() != 2) {
+                throw new IllegalArgumentException("\'reanchors\' must be a list of node pairs");
+            }
+
+            Node from = pairing.get(0);
+            Node to = pairing.get(1);
+
+            if (from == null || to == null) {
+                throw new IllegalArgumentException("\'reanchors\' must be a list of node pairs");
+            }
+
+            reanchorMap.put(from, to);
+        }
+
+        // clone nodes and populate copy map
+        for (Node node : nodes) {
+            if (node == null || reanchorMap.containsKey(node)) continue;
+            // nodes being reanchored from will NOT be cloned
+
+            NodeRefactorResult result = new NodeRefactorResult(node.getId());
+            try {
+                Node copy = copyLabels(node, db.createNode());
+
+                Map<String, Object> properties = node.getAllProperties();
```

Comment:
don't we have the code for that already? including skipping? if node we should probably extend the copyProperties method with the skip-list.

Reply:
The doCloneNodes() method has something similar, but it's processing this via a stream, which we can't use since we need to further process the results.

We also add the nodes to a map structure which doesn't look like is needed in the other method.

Reply:
Because of the above, I'm not going to reuse or combine the code. Maybe will look at it for a future refactor if needed.